doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -125,6 +125,15 @@ list of assertion commands is given in :ref:`Assertions`. The command
    That is, you have to give the full proof in one gulp, as a
    proof term (see Section :ref:`applyingtheorems`).
 
+   .. warning::
+
+      The use of this command is discouraged.  In particular, it is
+      known to not work in Proof General because this command must
+      immediately follow the command that opened the proof mode, but
+      Proof General inserts :cmd:`Unset` :flag:`Silent` before (see
+      `Proof General issue #498
+      <https://github.com/ProofGeneral/PG/issues/498>`_).
```

Comment:
```suggestion
      Use of this command is discouraged.  In particular, it
      doesn't work in Proof General because it must
      immediately follow the command that opened proof mode, but
      Proof General inserts :cmd:`Unset` :flag:`Silent` before it (see
      `Proof General issue #498
      <https://github.com/ProofGeneral/PG/issues/498>`_).
```

This doesn't fully explain why the command is discouraged--is there
a problem using the command in, say, CoqIDE?  Would it be worthwhile
to eventually relax the restriction (which sounds a little odd)?

Reply:
No there is no problem in using the command in CoqIDE as long you use it immediately after `Lemma` but it means you won't be able to replay the script in Proof General afterward. It is not deprecated yet because we do not provide an alternative that would satisfy everyone, but as soon as we do (the idea is `Lemma ... :=`, cf. coq/ceps#42), then it will. Therefore, there is no will to fix this command. I think that linking to the relevant issue is enough to have all the background information that one could hope for.